### ISSUE TITLE
Fade effect behind close diaglogue covers part of the window

### ISSUE THREAD

**Issue by AdrianDeWinter** [microsoft/terminal#11770]
### Windows Terminal version

1.112921.0

### Windows build number

19043.1348

### Steps to reproduce

Open Terminal with more tah one tab.
Set the Window to fullscreen.
Attempt to close the window.
The "Do you want to close all tabs?" dialogue appears

### Expected Behavior

The fade effect should cover the entire window behind the dialogue, not just a corner of it

### Actual Behavior

Only a part of the window gets faded to white, specifically about a quarter in the top left corner, of roughly the the same size as the default terminal window

**Comment by KalleOlaviNiemitalo** [comment]
This is a duplicate of <https://github.com/microsoft/terminal/issues/2380>, which was closed as a duplicate of <https://github.com/microsoft/microsoft-ui-xaml/issues/3577>.

(Personally, I'm more used to a workflow where I keep an issue open in the affected project and mark it blocked by the issue in the other project. Not sure how well GitHub supports that, though.)

**Comment by zadjii-msft** [comment]
More often than not, I usually do keep the issue on our side open, but also 🤷 That one's an OS-side bug so it's not like we'll need to ingest a new WinUI version to get the fix. When it gets ported to xaml islands and serviced, we should just magically stop hearing about this 😛 

/dup #2380
